tests/unit/authenticators/test-test.js

```diff
@@ -23,6 +23,14 @@ describe('Authenticators.Test', function() {
         done();
       });
     });
+
+    it('resolves with session data', function(done) {
+      this.authenticator.authenticate({ userId: 1, otherData: 'some-data' }).then(function(data) {
+        expect(true).to.be.true;
+        expect(data).to.eql({ userId: 1, otherData: 'some-data' });
+        done();
+      });
+    });
   });
```

Comment:
you can change that to 

```js
it('resolves with session data', function() {
  this.authenticator.authenticate({ userId: 1, otherData: 'some-data' }).then(function(data) {
    expect(data).to.eql({ userId: 1, otherData: 'some-data' });
  });
});
```

By returning the promise you get rid of the `done` callback. Also the first expect isn't actually needed.